index.bs

```diff
@@ -122,31 +122,24 @@ EmptyParams = { *text }
 <pre class="cddl local-cddl">
 Message = (
   CommandResponse //
+  ErrorResponse //
   Event
 )
 
-CommandResponse = (
+CommandResponse = {
   id: uint,
-  ResponseData,
-)
-
-ResponseData = (
-  Error //
-  CommandResult
-)
+  value: ResultData,
```

Comment:
```suggestion
  result: ResultData,
```

I see it was already `value` in the schema and this is just moving things around, but this seems like an unnecessary divergence from existing protocols and https://www.jsonrpc.org/specification#response_object. Happy to file an issue if you don't want to change it in this PR.

Reply:
This was designed to match WebDriver/HTTP. Let's discuss in an issue.

Reply:
I've opened https://github.com/w3c/webdriver-bidi/issues/58.